Include/cpython/code.h

```diff
@@ -62,7 +62,8 @@ typedef uint16_t _Py_CODEUNIT;
     PyObject *co_exceptiontable;   /* Byte string encoding exception handling  \
                                       table */                                 \
     int co_flags;                  /* CO_..., see below */                     \
-    int co_warmup;                 /* Warmup counter for quickening */         \
+    short co_warmup;                 /* Warmup counter for quickening */       \
+    short _co_linearray_entry_size;  /* Size of each entry in _co_linearray */ \
```

Comment:
Not a big deal since you were able to squeeze this in here, but we might want to consider making this the first byte of the line array (or a flag on the code object) if we end up getting rid of `co_warmup` later.

Reply:
I originally made it a bit in `co_flags` but I'm worried that might be considered an API change by some.